GCSViews/ConfigurationView/ConfigHWCompass2.cs

```diff
@@ -27,20 +27,20 @@ public class CompassDeviceInfo : DeviceInfo
             private string _orient;
             private bool _external;
 
-            public CompassDeviceInfo(int index, string ParamName, uint id) : base(index, ParamName, id)
+            public CompassDeviceInfo(int index, string ParamName, uint id, ConfigHWCompass2 compass2) : base(index, ParamName, id)

```

Comment:
i know its your first attempt, but ill just point out stuff

instead of passing the instance, i would have made the function static, so it can be used throughout without needing the instance version